modules/data.atmosphere/R/met.process.R

```diff
@@ -92,7 +92,16 @@ met.process <- function(site, input_met, start_date, end_date, model,
 
   on.exit(PEcAn.DB::db.close(con), add = TRUE)
   username <- ifelse(is.null(input_met$username), "pecan", input_met$username)
-  machine.host <- ifelse(host == "localhost" || host$name == "localhost", PEcAn.remote::fqdn(), host$name)
+
+  if (length(host) == 1 && !hasName(host, "name")) {
```

Comment:
```suggestion
  if (length(host) == 1 && !("name" %in% names(host))) {
```

Would help bypass the `make check` errors without changing the `NAMESPACE`